networking/spoke-BU0001A0008.bicep

```diff
@@ -147,7 +147,7 @@ resource nsgAppGwSubnet 'Microsoft.Network/networkSecurityGroups@2021-05-01' = {
           description: 'Allow Azure Control Plane in. (https://docs.microsoft.com/azure/application-gateway/configuration-infrastructure#network-security-groups)'
           protocol: '*'
           sourcePortRange: '*'
-          sourceAddressPrefix: '*'
+          sourceAddressPrefix: 'GatewayManager'
```

Comment:
Please update the `AllowHealthProbesInbound` rule right below this one, line 166.

```json
destinationAddressPrefix: 'VirtualNetwork'
```

to

```json
destinationAddressPrefix: '*'
```

(Sorry I can't do a GitHub suggestion -- you can only do that on changes near other changes -- and apparently this isn't "near enough")

Without this change as well, your proposal will not deploy.  This final end state matches the docs content as well.